governance/roles.md

```diff
@@ -92,23 +93,37 @@ TLs are assigned by CNCF Technical Oversight Committee
 
 ## Role of project leads
 
-Project Leads will lead larger streams of work that require sustained
-effort and coordination.  In some cases, these projects remain ongoing.
+Project Leads lead specific projects where a deliverable or set of deliverables
```

Comment:
I don't think we should encourage a single issue to ever have a set of deliverables.